src/components/Inventory.js

```diff
@@ -2,17 +2,28 @@ import React from "react";
 import DataTable from "./DataTable";
 
 class Inventory extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      api_rows: []
+    };
+  }
+
+  async componentDidMount() {
+    const response = await fetch("http://localhost:3001/inventory");
+    const json = await response.json();
+    this.setState({ api_rows: json });
+  }
+
   render() {
-    const headings = ["Item", "Count", "Price", "Cost", "Profit"];
+    const headings = ["Item", "Count", "Price", "Sold"];
 
-    const rows = [
-      ["Heart Shirt", 20, 10, "$9.99", "$0.01"],
-      ["Skull Shirt", 50, 20, "$9.99", "$10.01"]
-    ];
     return (
-      <div className="Inventory">
-        <h1>Inventory</h1>
-        <DataTable headings={headings} rows={rows} />
+      <div>
+        <div className="Inventory">
+          <h1>Inventory</h1>
```

Comment:
look in the order component to determine the class name for headers and give the same class name to this h1 tag